AnimatedTextInput/Classes/AnimatedTextInput.swift

```diff
@@ -170,7 +170,7 @@ open class AnimatedTextInput: UIControl {
         }
     }
 
-    fileprivate let lineView = AnimatedLine()
+    public let lineView = AnimatedLine()
```

Comment:
Although this works, I would prefer having 2 new methods than making the line public. Something like:
`func hideLine(animated: Bool)`
Internally this would either call `isHidden` or `fillLine(with: UIColor.clear)`.

`func showLine(animated: Bool)`
Internally this would either call `isHidden = false` or `fillLine(with: style.lineColor)`

Reply:
thanks